src/components/NavBar/NavBar.jsx

```diff
@@ -1,21 +1,58 @@
 import {Add, Home} from '@mui/icons-material';
+import {useState} from 'react';
 import {NavLink} from 'react-router-dom';
+import styled from 'styled-components';
 
 import NavContainer from './NavBar.styled';
 
 function NavBar() {
+	const [active, setActive] = useState(0);
 	return (
-		<NavContainer>
-			<NavLink to="/" data-testid="navHome">
-				<Home />
-				<div></div>
-			</NavLink>
-			<NavLink to="/create" data-testid="navCreate">
-				<Add />
-				<div></div>
-			</NavLink>
-		</NavContainer>
+		<>
+			{/* <ContainerDiv /> */}
+			<NavContainer active={active}>
+				<ul>
+					<li
+						onClick={() => {
+							setActive(1);
```

Comment:
Just a Suggestion, to set active status with numbers instead of names is not symantic. The better ways would be to chose a good naming instead of a number.